paywall/next.config.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs')
+const { join } = require('path')
+const { promisify } = require('util')
+const withSourceMaps = require('@zeit/next-source-maps')
+
+const copyFile = promisify(fs.copyFile)
+
+module.exports = withSourceMaps({
+  publicRuntimeConfig: {
+    unlockEnv: process.env.UNLOCK_ENV || 'dev',
+    httpProvider: process.env.HTTP_PROVIDER || '127.0.0.1',
+    readOnlyProvider: process.env.READ_ONLY_PROVIDER,
+    locksmithHost: process.env.LOCKSMITH_HOST,
```

Comment:
I don't think will be needed for the paywall, not show stopping and can be removed later

Reply:
the locksmithHost yes? I agree. This is mainly about getting the existing code over, there will be a lot of pruning once things are here.